src/main/java/com/volmit/iris/engine/IrisExecutionEnvironment.java

```diff
@@ -54,9 +55,13 @@ public IrisScriptingAPI getAPI() {
     }
 
     public void execute(String script) {
-        Iris.debug("Execute Script (void) " + C.DARK_GREEN + script);
+        execute(getEngine().getData().getScriptLoader().load(script));
+    }
+
+    public void execute(IrisScript script) {
```

Comment:
Again, use script.getLoadKey, dont duplicate methods to reduce code in the command itself. Remove this method and use the old one with your command parameter

Reply:
Seven